server-adapters/nestjs/src/index.ts

```
@@ -0,0 +1,145 @@
+  }
+
+  async sendResponse(route: ServerRoute, response: unknown, result: unknown, request?: unknown): Promise<void> {
+    return this.platformAdapter.sendResponse(route, response, result, request);
+  }
+
+  async registerRoute(app: INestApplication, route: ServerRoute, options: { prefix?: string }): Promise<void> {
+    return this.platformAdapter.registerRoute(route, options);
+  }
+
+  registerContextMiddleware(): void {
+    return this.platformAdapter.registerContextMiddleware();
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)